.github/workflows/build-package.yml

```
@@ -0,0 +1,60 @@
+name: Build Package
+
+env:
+  MAIN_BRANCH: "all-citus"
+  PACKAGE_CLOUD_REPO_NAME: "sample"
+
+on:
+  push:
+    branches: "**"
+
+  workflow_dispatch:
+
+jobs:
+  build_package:
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        platform: [ el/7,
+                    el/8,
+                    ol/7,
+                    debian/stretch,
+                    debian/buster,
+                    ubuntu/xenial,
+                    ubuntu/bionic,
+                    ubuntu/focal ]
```

```suggestion
        platform:
          - el/7,
          - el/8,
          - ol/7,
          - debian/stretch,
          - debian/buster,
          - ubuntu/xenial,
          - ubuntu/bionic,
          - ubuntu/focal
```

IMO arrays are easier to read when we drop `[]` and use yaml features for lists instead.

---

Fixed